bool isStarted
---
sensor_msgs/LaserScan laser
